packages/applications-service-api/src/repositories/document.backoffice.repository.js

```diff
@@ -0,0 +1,76 @@
+const { createPrismaClient } = require('../lib/prisma');
+const { Prisma } = require('@prisma/client');
+
+const prismaClient = createPrismaClient();
+
+const getFilters = (caseReference) => {
+	const sql = Prisma.sql`
+        SELECT DISTINCT(stage), filter1, count(id) as total
+        FROM Document
+        WHERE caseRef = ${caseReference}
+          AND (stage is not null and stage <> 'draft')
+          AND filter1 is not null
+        GROUP BY stage, filter1`;
+
+	return prismaClient.$queryRaw(sql);
+};
+
+const getDocuments = async (query) => {
+	let whereClause = {
+		AND: [
+			{ caseRef: query.caseReference },
+			{
+				AND: [{ stage: { not: { equals: null } } }, { stage: { not: { equals: 'draft' } } }]
+			}
+		]
+	};
+
+	if (query.searchTerm) {
+		whereClause['AND'].push({
+			OR: [
+				{ description: { contains: query.searchTerm } },
+				{ author: { contains: query.searchTerm } },
+				{ representative: { contains: query.searchTerm } },
+				{ mime: { contains: query.searchTerm } }
```

Comment:
I never realised we search on mime, but you are correct we currently do, seems really odd, wonder if path would be more useful.

Reply:
Yeah I carried this over from the original /v1/documents endpoint, although in retrospect it doesn't seem very useful to search on `mime`, unless we have a usecase for searching by file type (word, excel, pdf, etc). Even then that need would probably be better served by a filter.

Happy to drop it, as the more fields you search across, the fuzzier the results become.